src/storage/tests/table_test.cc

```diff
@@ -100,99 +100,77 @@ class HustleTableTest : public testing::Test {
 }
 
 TEST_F(HustleTableTest, OneBlockTable) {
   DBTable table("table", schema, BLOCK_SIZE);
-
-  table.insert_record((uint8_t *)record_string.data(), byte_widths);
-
+  table.InsertRecord((uint8_t *)record_string.data(), byte_widths);
   EXPECT_EQ(table.get_num_blocks(), 1);
 }
 
 TEST_F(HustleTableTest, OneBlockArray) {
   DBTable table("table", schema, BLOCK_SIZE);
-
-  table.insert_records(column_data);
-
+  table.InsertRecords(column_data);
   EXPECT_EQ(table.get_num_blocks(), 1);
 }
 
 TEST_F(HustleTableTest, TwoBlockArray) {
   DBTable table("table", schema, BLOCK_SIZE);
-
   // Inserting three records 14 times. This fits into one block.
   for (int i = 0; i < 14; i++) {
-    table.insert_records(column_data);
+    table.InsertRecords(column_data);
   }
-
   // Bulk inserting again should allocate a new block
-  table.insert_records(column_data);
-
+  table.InsertRecords(column_data);
   EXPECT_EQ(table.get_num_blocks(), 2);
 }
 
 TEST_F(HustleTableTest, TwoBlockTable) {
   DBTable table("table", schema, BLOCK_SIZE);
-
   // With 1 KB block size, we can store 8 copies of the first record in one
   // block.
   for (int i = 0; i < 8; i++) {
-    table.insert_record((uint8_t *)record_string.data(), byte_widths);
+    table.InsertRecord((uint8_t *)record_string.data(), byte_widths);
   }
-
   // The first block cannot hold a 9th copy of the first record, so we must
   // create a new block.
-  table.insert_record((uint8_t *)record_string.data(), byte_widths);
-
+  table.InsertRecord((uint8_t *)record_string.data(), byte_widths);
   EXPECT_EQ(table.get_num_blocks(), 2);
 }
 
 TEST_F(HustleTableTest, TableIO) {
   DBTable table("table", schema, BLOCK_SIZE);
-
-  table.insert_records(column_data);
-  table.insert_record((uint8_t *)record_string.data(), byte_widths);
-  table.insert_records(column_data);
-
+  table.InsertRecords(column_data);
+  table.InsertRecord((uint8_t *)record_string.data(), byte_widths);
+  table.InsertRecords(column_data);
   for (int i = 0; i < 8; i++) {
-    table.insert_record((uint8_t *)record_string.data(), byte_widths);
+    table.InsertRecord((uint8_t *)record_string.data(), byte_widths);
   }
-
   // The first block cannot hold a 9th copy of the first record, so we must
   // create a new block.
-  table.insert_record((uint8_t *)record_string.data(), byte_widths);
-
+  table.InsertRecord((uint8_t *)record_string.data(), byte_widths);
   write_to_file("table.hsl", table);
-
   auto table_from_file = read_from_file("table.hsl");
-
-  EXPECT_TRUE(table_from_file->get_block(0)->get_valid_column()->Equals(
-      *table.get_block(0)->get_valid_column()));
-  EXPECT_TRUE(table_from_file->get_block(0)->get_records()->Equals(
-      *table.get_block(0)->get_records()));
-  EXPECT_TRUE(table_from_file->get_block(1)->get_records()->Equals(
-      *table.get_block(1)->get_records()));
+  EXPECT_TRUE(
+      table_from_file->get_block(0)->get_valid_column()->Equals(*table.get_block(0)->get_valid_column()));
```

Comment:
Is this <=80 chars? it looks wider, Github should have a line at 80 chars to make this easy to check :)